### PR TITLE
feat: add guid nodeids

### PR DESCRIPTION
**Pull request by Kantiran91**
Add the converter for the guid nodeids. So the issues with #465 and #430 should be fixed
@GoetzGoerisch Can you validate this with an server?

### CODE REVIEW

File: OpcUaClient/Converter/ModelNodeIdToUaNodeId.cpp [hunk 1]
```diff
@@ -14,49 +15,88 @@
...
+    else if(hex >= 'A' && hex <= 'F')
+        return hex - 'A' + 10;
+    else
+	    LOG(ERROR) << "Invalid char in Guid String";
+        throw std::invalid_argument("Invalid char in Guid String");
+ 
+}
+
+//Convert an hex string to a UA_GUID
+UA_Guid hex_to_UA_Guid(const char* hex) {
```
Comment (Kantiran91) [inline OpcUaClient/Converter/ModelNodeIdToUaNodeId.cpp:37]: @ccvca I think this method is very generic. Is there an better place for this? e.g. open62541Cpp or the open62541 Lib itself?

Comment (ccvca) [reply OpcUaClient/Converter/ModelNodeIdToUaNodeId.cpp:37]: GUID parsing already part of open62541 https://github.com/open62541/open62541/blob/1f47491d0d9a406630a9de1c1d21eb1987885497/include/open62541/types.h#LL324C3-L324C3

I think the whole parsing (including Bytestring) can be replaces with this function (Added Mar 2020, 6 months after the EMO 2019 ):
https://github.com/open62541/open62541/blob/1f47491d0d9a406630a9de1c1d21eb1987885497/include/open62541/types.h#L423

Comment (Kantiran91) [reply OpcUaClient/Converter/ModelNodeIdToUaNodeId.cpp:37]: thank you - haven´t seen I add this to the branch

Comment (Kantiran91) [reply OpcUaClient/Converter/ModelNodeIdToUaNodeId.cpp]: fixed

**Comment by GoetzGoerisch** [comment]
Great!
Feedback requrested from @moos-daniel

**Pull request merged**